aggregator_api/src/routes.rs

```diff
@@ -321,7 +323,8 @@ pub(super) async fn put_global_hpke_config<C: Clock>(
         req.kem_id.unwrap_or(HpkeKemId::X25519HkdfSha256),
         req.kdf_id.unwrap_or(HpkeKdfId::HkdfSha256),
         req.aead_id.unwrap_or(HpkeAeadId::Aes128Gcm),
-    );
+    )
+    .unwrap();
```

Comment:
We should propagate this error instead of unwrapping it, since the algorithms could be unsupported if the operator makes a mistake.